session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -154,7 +167,12 @@ public synchronized void close() throws IoTDBSessionException {
     }
   }
 
-  public synchronized TSExecuteBatchStatementResp insertBatch(RowBatch rowBatch)
```

Comment:
Why we no longer need the synchronized.

Reply:
We could see that all the variables in the method is local, so no concurrent problem without synchronized